pom.xml

```diff
@@ -113,6 +113,16 @@
       <artifactId>org.everit.json.schema</artifactId>
       <version>1.12.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>mod-pubsub-client</artifactId>
+      <version>1.1.5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>util</artifactId>
+      <version>${rmb.version}</version>
```

Comment:
Have you checked that it's not getting in the way of PubSub client dependency and the client doesn't throw NoClassDefFoundError when it tries to register or send events?

Reply:
No, of course it doesn't if tests are passing.. Never mind